### ISSUE TITLE
What is the best way to load the large stackoverflow.com database?

### ISSUE THREAD

**Issue by arvindshmicrosoft** [phil-scott-78/dotnet-soddi#5]
Thank you for the great work here!

I was trying to load the large stackoverflow.com database. Due to the large size, the stackoverflow.com dump from archive.org is comprised of several files:
- https://archive.org/download/stackexchange/stackoverflow.com-Badges.7z
- https://archive.org/download/stackexchange/stackoverflow.com-Comments.7z
- https://archive.org/download/stackexchange/stackoverflow.com-PostHistory.7z
- https://archive.org/download/stackexchange/stackoverflow.com-PostLinks.7z
- https://archive.org/download/stackexchange/stackoverflow.com-Posts.7z
- https://archive.org/download/stackexchange/stackoverflow.com-Tags.7z
- https://archive.org/download/stackexchange/stackoverflow.com-Users.7z
- https://archive.org/download/stackexchange/stackoverflow.com-Votes.7z

I am unsure how to proceed with loading these, when the files were downloaded previously. The first file works fine:
`soddi import stackoverflow.com-Comments.7z -d "stackoverflow" -c "<REDACTED>"`

When attempting to incrementally do the next file (let's say, PostLinks), the `Schema already exists in database` error is raised.

I also tried to use the "substring" approach:
`soddi import stackoverflow.com -d "stackoverflow" -c "<REDACTED>"`

Unfortunately, that errors out as well:
`Error: Could not find archive stackoverflow.com`

I'm hoping this is operator error, but maybe there is something else at play?

**Comment by phil-scott-78** [comment]
Ok, it took me a minute to get my mind back in order from when I wrote this. For the multiple part files the logic I used was to put them in a folder and then point towards that folder.


e.g. given a folder named stackoverflow with those files this command should work. I'll get the documentation updated to include this example, and I'll also maybe try and figure out a better way to improve this experience

 ```
soddi import stackoverflow  -d "stackoverflow" -c "<REDACTED>"
```
